jcommune-service/src/main/java/org/jtalks/jcommune/service/nontransactional/NotificationService.java

```diff
@@ -85,15 +85,15 @@ public void subscribedEntityChanged(SubscriptionAwareEntity entity) {
     /**
      * Overload for skipping topic subscribers
      *
-     * @param entity
-     * @param topicSubscribers
+     * @param entity changed subscribed entity.
+     * @param excludeFromNotification a collection of subscribers who are excluded from a notification.
      */
-    public void subscribedEntityChanged(SubscriptionAwareEntity entity, Collection<JCUser> topicSubscribers) {
+    public void subscribedEntityChanged(SubscriptionAwareEntity entity, Collection<JCUser> excludeFromNotification) {
         Collection<JCUser> subscribers = subscriptionService.getAllowedSubscribers(entity);
         filterSubscribers(subscribers, entity);
 
         for (JCUser user : subscribers) {
```

Comment:
Just figured.. Why don't we do `subscribers.removeAll(excludeFromNotification)`?